address/address.py

```diff
@@ -1,11 +1,14 @@
 # Meant to parse out address lines, minus city,state,zip into a usable dict for address matching
 # Ignores periods and commas, because no one cares.
 
+from __future__ import absolute_import
+from __future__ import print_function
 import re
 import csv
 import os
-import dstk
+from . import dstk
```

Comment:
This is failing in python 3.7 with error: 
    from . import dstk
ImportError: cannot import name 'dstk' from '__main__'
can you pls verify it , thank you.